pkg/scheduler/internal/cache/cache.go

```diff
@@ -691,17 +675,17 @@ func (cache *cacheImpl) addNodeImageStates(node *v1.Node, nodeInfo *framework.No
 			// update the entry in imageStates
```

Comment:
s/imageStates/ImageStateSummary

Reply:
there is no need change to be ImageStateSummary,  imageStates just the field of cache struct

Reply:
mark this as resolved as well pls.